### PR TITLE
9.0 end of pipe yti

### PR DESCRIPTION
**Pull request by tivisse**
TASK : https://www.odoo.com/web#view_type=form&model=project.task&action=333&active_id=248
PAD : https://pad.odoo.com/p/r.CSbwt3LDi0zr0WU4

### CODE REVIEW

File: addons/hr_recruitment/static/description/index.html [hunk 1]
```diff
@@ -71,7 +68,7 @@ <h3 class="oe_slogan">Define your own stages and interviewers</h3>
          <h2 class="oe_slogan">Streamline Your Recruitment Process</h2>
          <h3 class="oe_slogan">Index resumes, track applicants, search profiles</h3>
         <div class="oe_span6">
-            <img class="oe_picture oe_screenshot" src="hr_recruitment.png">
+            <img class="oe_picture oe_screenshot" src="">
```
Comment (tde-banana-odoo) [inline addons/hr_recruitment/static/description/index.html]: Usefull to keep an image without src ?

File: addons/l10n_fr_hr_payroll/views/report_l10nfrfichepaye.xml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+q<?xml version="1.0" encoding="utf-8"?>
```
Comment (tde-banana-odoo) [inline addons/l10n_fr_hr_payroll/views/report_l10nfrfichepaye.xml]: This change does not seem necessary :) .

File: addons/project/project_view.xml [hunk 3]
```diff
@@ -396,8 +395,8 @@
                     <field name="planned_hours" invisible="context.get('set_visible',False)" groups="project.group_time_work_estimation_tasks"/>
                     <field name="date_deadline" invisible="context.get('deadline_visible',True)"/>
                     <field name="stage_id" invisible="context.get('set_visible',False)"/>
-                    <field name="date_start" groups="project.group_time_work_estimation_tasks"/>
-                    <field name="date_end" groups="project.group_time_work_estimation_tasks"/>
+                    <field name="date_start" groups="project.group_time_work_estimation_tasks" invisible="1"/>
```
Comment (tde-banana-odoo) [inline addons/project/project_view.xml]: group + invisible ?

File: addons/l10n_fr_hr_payroll/views/report_l10nfrfichepaye.xml (hunk 2 above)
Comment (tivisse) [inline addons/l10n_fr_hr_payroll/views/report_l10nfrfichepaye.xml]: Indeed, that's why it was red ;)

**Pull request merged**